2-structured-data/config/settings.yml.dist

```diff
@@ -2,15 +2,22 @@
 google_client_id:      YOUR_CLIENT_ID
 google_client_secret:  YOUR_CLIENT_SECRET
 google_project_id:     YOUR_PROJECT_ID
+cloudsql_connection_name: "INSTANCE_CONNECTION_NAME"
 
-# options are "cloudsql", "mongodb", or "datastore"
-bookshelf_backend: cloudsql
+# options are "cloudsql-mysql", "cloudsql-postgres", "mongodb", or "datastore"
+bookshelf_backend: cloudsql-mysql
 
-# configure CloudSQL backend
-mysql_dsn: "mysql:host=IP_ADDRESS;port=3306;dbname=YOUR_DB_NAME"
+# configure CloudSQL for MySQL backend
+mysql_dsn_local: "mysql:host=127.0.0.1;port=3306;dbname=YOUR_DB_NAME"
+mysql_dsn_deployed: "mysql:unix_socket=/mysql/INSTANCE_CONNECTION_NAME;dbname=YOUR_DB_NAME"
 mysql_user: YOUR_MYSQL_USER
 mysql_password: YOUR_MYSQL_PASSWORD
-cloudsql_connection_name: "INSTANCE_CONNECTION_NAME"
+
+# configure CloudSQL for Postgres backend
+postgres_dsn_local: "pgsql:host=127.0.0.1;port=5432;dbname=YOUR_DB_NAME"
+postgres_dsn_deployed: "pgsql:host=/cloudsql/INSTANCE_CONNECTION_NAME;dbname=YOUR_DB_NAME"
```

Comment:
To simplify things further, instead of having `*_dsn_local` and `*_dsn_deployed`, just make a `sql_dbname` and automatically create the DSNs from `cloudsql_connection_name`, `bookshelf_backend`, and whether or not we're on app engine.

Thoughts?